lib/teleterm/clusters/cluster_databases.go

```diff
@@ -60,7 +60,9 @@ func (c *Cluster) GetDatabase(ctx context.Context, dbURI string) (*Database, err
 }
 
 // GetDatabases returns databases
-func (c *Cluster) GetAllDatabases(ctx context.Context) ([]Database, error) {
+// TODO(ravicious): Remove this method in favor of fetching a single database in GetDatabase.
+// https://github.com/gravitational/teleport/pull/14690#discussion_r927720600
+func (c *Cluster) getAllDatabases(ctx context.Context) ([]Database, error) {
```

Comment:
I'd have removed it already but [I'm trying to establish what's the efficient way to do that](https://gravitational.slack.com/archives/C0DF0TPMY/p1678276982792309).

I'll probably submit another PR if I get to that.